### DISCUSSION TITLE
Allow additional kernel command line options

### DISCUSSION THREAD

**Discussion by herrernst** [johang/sd-card-images#98] [Ideas]
I see the kernel args are hardcoded in https://github.com/johang/sd-card-images/blob/master/2nd-stage-files/pre-2nd-stage-files-arm64/etc/initramfs/post-update.d/zz-update-uimg#L22

I have added `rootflags=noatime` for me to prevent wear-out of my sd-card. Similarly, one maybe wants to have `rootflags=discard` added if one has an SSD with reliable TRIM feature. 
Maybe optional kernel command line args could be read from a config file? Armbian has `uEnv.txt`, but it shouldn't be to complicated or rely on forked packages. Maybe you have an idea for a simple solution? Thanks

By the way, great project, thank you! I like the minimalist approach very much!

**Comment by johang** [comment]
I think this is more or less already possible.

Put something like `extrabootargs=rootflags=noatime` in the file `uboot.env` in the FAT partition then add `\${extrabootargs}` to the line you showed.

**Reply by herrernst** [reply]
Thanks for your answer, unfortunately this had no effect for me (the options in `uboot.env` didn't get applied).

**Reply by johang** [reply]
Maybe that's a platform specific feature in U-Boot then.
